samples/Plugin.Maui.Audio.Sample/ViewModels/BitDepthViewModel.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace Plugin.Maui.Audio.Sample.ViewModels;
+
+public class BitDepthViewModel
```

Comment:
As I mentioned in the view model changes, I don't think we need these wrapper classes